### PR TITLE
#5 Give visual feedback when clicking a copy button

### PR DESCRIPTION
**Pull request by Baeliale**
#5 Give visual feedback when clicking a copy button through a snackbar notification

### CODE REVIEW

File: containers/account/AccountDetails.tsx [hunk 1]
```diff
@@ -74,6 +75,14 @@ export const AccountDetails: FC<{
...
   }, [serviceClient, account])
 
+  const [copyNoteToggle, setCopyNoteToggle] = useState<boolean>(false)
+  const [copyNoteText, setCopyNoteText] = useState<string>("")
+  const copyNotification = (text: string) => {
+    setCopyNoteToggle(true)
+    setCopyNoteText(text)
+    setTimeout(() => setCopyNoteToggle(false), 5000)
+  }
+
```
Comment (corbifex) [inline containers/account/AccountDetails.tsx]: - Can you remove the toggle state and use the text state as the toggle.

File: containers/account/AccountDetails.tsx [hunk 2]
```diff
@@ -74,6 +75,14 @@ export const AccountDetails: FC<{
     }
   }, [serviceClient, account])
 
+  const [copyNoteToggle, setCopyNoteToggle] = useState<boolean>(false)
+  const [copyNoteText, setCopyNoteText] = useState<string>("")
+  const copyNotification = (text: string) => {
+    setCopyNoteToggle(true)
+    setCopyNoteText(text)
+    setTimeout(() => setCopyNoteToggle(false), 5000)
```
Comment (corbifex) [inline containers/account/AccountDetails.tsx]: the setTimeout doesn't clear on a additional click, when you click an other copy button within 5 seconds the snackbar hides really fast

File: containers/account/AccountDetails.tsx [hunk 3]
```diff
@@ -116,7 +125,7 @@ export const AccountDetails: FC<{
             <CopyToClipboard
               text={account?.summary?.username || account?.summary?.address}
             >
-              <IconButton className=" focus:text-accentPrimary text-surfacePrimaryDark ">
+              <IconButton onClick={copyNotification.bind(this, "Username copied")} className=" focus:text-accentPrimary text-surfacePrimaryDark ">
```
Comment (corbifex) [inline containers/account/AccountDetails.tsx]: Change bind notation for `onClick={() => copyNotification("Username copied")}`

**Review by corbifex — APPROVED** [review]
LGTM

**Review by jurrem — APPROVED** [review]

**Pull request merged**